### PR TITLE
Add JSON `extra` field

### LINKED ISSUE #50: Add `extras` field in model objects
To help make it easier for frontend devs to make changes, add an "extras" field that can contain arbitrary JSON and will be stored by the server. 

Anything in the extras field that seems like it will stay can be moved to an actual field, of course.

### PR DESCRIPTION
**Pull request by ryansb**
Closes #50

### CODE REVIEW

**Comment by ryansb** [comment]
Also, this works on SQLite even though it lacks JSON support, since [sqlalchemy_utils](https://sqlalchemy-utils.readthedocs.org/en/latest/data_types.html#module-sqlalchemy_utils.types.json) has a JSON type that's db-agnostic.

File: server/mediapublic/models.py [hunk 1]
```diff
@@ -99,13 +107,10 @@ def reqkeys(cls):
         return keys
 
     def to_dict(self):
-        return {
-            'id': str(self.id),
-            'creation_datetime': str(self.creation_datetime),
-        }
```
Comment (thequbit) [inline server/mediapublic/models.py:105]: Why did you get rid of the `creation_datetime` field?

Comment (thequbit) [inline server/mediapublic/models.py:105]: err, I see - it's now included in the to_dict() in the little builder, yes?

Comment (ryansb) [inline server/mediapublic/models.py:105]: Ah, I didn't intend to remove that.

**Pull request merged**